### PR TITLE
chore: [TECH-862] [TECH-1409] fix cypress tests

### PR DESCRIPTION
**Pull request by JoakimSM**
Fixes Cypress tests listed in [TECH-862](https://dhis2.atlassian.net/browse/TECH-862) and other failing tests previously listed in our wiki.

### CODE REVIEW

File: cypress/integration/SearchPage.feature [hunk 1]
```diff
@@ -1,248 +1,217 @@
...
-#    Given you are in the search page with the Child Programme being preselected from the url
-#    And you expand the attributes search area
-#    When you fill in the first and last name with values that will return results
-#    And you click search
-#    And you can see the first page of the results
-#    When you click the fallback search button
-#    Then you see the attributes search area being expanded
-#    And that first and last name are prefilled
-
-  Scenario: Fallback search excludes attributes that are not relevant in the tetype scope
```
Comment (simonadomnisoru) [inline cypress/integration/SearchPage.feature:222]: Hey,
Any reason why the scenarios `Fallback search excludes attributes that are not relevant in the tetype scope` and `Fallback search with results and navigation back to the program show` were deleted? They were not disabled beforehand which means that in theory, they are valid tests. 
Thank you!

File: cypress/integration/SearchPage.feature [hunk 2]
```diff
@@ -1,248 +1,217 @@
...
-  Scenario: Fallback search with results
-    Given you are in the search page with the Child Programme being preselected from the url
-    And you expand the attributes search area
-    When you fill in the first and last name with values that will return results
-    And you click search
-    And you can see the first page of the results
-    When you click the fallback search button
-    Then you stay in the same page with all program results
-
-# TODO - Commenting out this test and creating a new tech-ticket (TECH-866)
```
Comment (simonadomnisoru) [inline cypress/integration/SearchPage.feature:210]: The ticket `TECH-866` is still open. Should we either close that ticket or try to find a solution to it?

File: cypress/integration/AllAccessibleRecordsPage.feature [hunk 3]
```diff
@@ -24,11 +24,3 @@ Feature: the user interacts with the "view all records regardless of OrgUnit"-pa
   Scenario: the working list will hydrate when updated by the URL
     When the user navigates to /#/?programId=VBqh0ynB2wv&all
     Then the working list should be displayed
-
-#  Scenario: The user can edit an event without selecting OrgUnit
```
Comment (simonadomnisoru) [inline cypress/integration/AllAccessibleRecordsPage.feature:28]: Hey,
Any reason why this scenario was removed? 
Thanks!

Comment (JoakimSM) [reply cypress/integration/AllAccessibleRecordsPage.feature:28]: Good catch. Re-added, fixed and enabled.

File: cypress/integration/SearchPage.feature (hunk 2 above)
Comment (JoakimSM) [reply cypress/integration/SearchPage.feature:210]: Thanks, ticket closed. The test scenario is no longer valid (the fallback workflow has changed)

**Review by simonadomnisoru — APPROVED** [review]

**Pull request merged**